src/components/status/status.scss

```
@@ -0,0 +1,62 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-status-gap: var(--sbb-spacing-fixed-1x);
+  --sbb-status-icon-color: var(--sbb-color-iron-default);
+  --sbb-status-icon-line-height: var(--sbb-font-size-text-s);
```

Maybe mark as an internal var (underscore prefix) as it is only used internally and from outside you can not really overwrite the font-size.
```suggestion
  --_sbb-status-icon-font-size: var(--sbb-font-size-text-s);
```